packages/main/src/Popover.ts

```diff
@@ -64,28 +64,17 @@ type CalculatedPlacement = {
  * or selects an action within the popover. You can prevent this with the
  * <code>modal</code> property.
```

Comment:
there are still properties with `@name` tag

Also `@param` tag should has type for every param (getArrowPosition method)